src/main/java/org/gridsuite/merge/orchestrator/server/repositories/MergeEntityKey.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.merge.orchestrator.server.repositories;
+
+import org.springframework.data.cassandra.core.cql.PrimaryKeyType;
+import org.springframework.data.cassandra.core.mapping.PrimaryKeyClass;
+import org.springframework.data.cassandra.core.mapping.PrimaryKeyColumn;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com
+ */
+@PrimaryKeyClass
+public class MergeEntityKey implements Serializable {
+
+    @PrimaryKeyColumn(ordinal = 0, type = PrimaryKeyType.PARTITIONED)
+    private String process;
+
+    @PrimaryKeyColumn(ordinal = 1, type = PrimaryKeyType.CLUSTERED)
+    private LocalDateTime date;
+
+    public MergeEntityKey(String process, LocalDateTime date) {
+        this.process = process;
+        this.date = date;
+    }
+
+    public String getProcess() {
```

Comment:
use Lombok @getter instead of writing getters  (as used in dto/CaseInfos)?

Reply:
Done